docs/scripts/buildApiUtils.ts

```diff
@@ -24,44 +24,6 @@ function getMuiName(name: string) {
   return `Mui${name.replace('Unstyled', '').replace('Styled', '')}`;
 }
 
-const componentPackageMapping = {
```

Comment:
This is not needed since the package name (eg. `mui-material`) is not in the URL anymore.